diff --git a/node_modules/@727-ventures/typechain-types/dist/src/query.d.ts b/node_modules/@727-ventures/typechain-types/dist/src/query.d.ts
index dc837a5..1c74218 100644
--- a/node_modules/@727-ventures/typechain-types/dist/src/query.d.ts
+++ b/node_modules/@727-ventures/typechain-types/dist/src/query.d.ts
@@ -1,12 +1,12 @@
 import type { ContractPromise } from "@polkadot/api-contract";
 import type { AnyJson } from '@polkadot/types-codec/types';
 import type { RequestArgumentType, GasLimitAndValue } from './types';
-import { Weight } from '@polkadot/types/interfaces';
+import { WeightV2 } from '@polkadot/types/interfaces';
 import { ApiPromise } from "@polkadot/api";
 type QueryReturnType<T> = {
     value: T;
-    gasConsumed: Weight;
-    gasRequired: Weight;
+    gasConsumed: WeightV2;
+    gasRequired: WeightV2;
 };
 export type { QueryReturnType, };
 export { _genValidGasLimitAndValue, };
@@ -25,8 +25,8 @@ export declare function queryOkJSON<T>(api: ApiPromise, nativeContract: Contract
  */
 export declare function queryOutput(api: ApiPromise, nativeContract: ContractPromise, callerAddress: string, title: string, args?: readonly RequestArgumentType[], gasLimitAndValue?: GasLimitAndValue): Promise<{
     output: import("@polkadot/types-codec/types").Codec;
-    gasConsumed: Weight;
-    gasRequired: Weight;
+    gasConsumed: WeightV2;
+    gasRequired: WeightV2;
 }>;
 declare function _genValidGasLimitAndValue(api: ApiPromise, gasLimitAndValue?: GasLimitAndValue): Promise<GasLimitAndValue>;
 export declare function handleReturnType(result: any, typeDescription: any): any;
diff --git a/node_modules/@727-ventures/typechain-types/dist/src/tx.js b/node_modules/@727-ventures/typechain-types/dist/src/tx.js
index e15b609..cd5f6ae 100644
--- a/node_modules/@727-ventures/typechain-types/dist/src/tx.js
+++ b/node_modules/@727-ventures/typechain-types/dist/src/tx.js
@@ -116,7 +116,7 @@ function _signAndSend(registry, extrinsic, signer, eventHandler) {
                         if (result.status.isInBlock) {
                             actionStatus.blockHash = result.status.asInBlock.toHex();
                         }
-                        if (result.status.isFinalized || result.status.isInBlock) {
+                        if (result.status.isFinalized) {
                             actionStatus.events = eventHandler(result.events);
                             result.events
                                 .filter(function (_a) {
